nautobot/dcim/api/serializers.py

```diff
@@ -1025,6 +1025,16 @@ def validate(self, data):
                 )
 
         # Validate many-to-many VLAN assignments
+        mode = data.get("mode", getattr(self.instance, "mode", None))
+        has_tagged_vlans = data.get("tagged_vlans", False)
+        if not has_tagged_vlans and self.instance and self.instance.tagged_vlans.exists():
```

Comment:
L1030 is doing a boolean check against `self.instance` but L1028 is not. Checking to see if this be a problem when creating a new interface

Reply:
Serializer `instance` is initialized to `None` so this is good.